apps/bbsync/mixins.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+
+from .constants import SYNC_TO_BZ
+from .exceptions import UnsavableModelError
+
+
+class BugzillaSyncMixin(models.Model):
+    """
+    mixin for syncing the model to the Bugzilla
+
+    the sync between Bugzilla and the internal DB is done in a way
+    that the model is first stored to the Bugzilla and then fetched
+    back from there to make sure that the timestaps are kept in sync
+    (otherwise there might be a few seconds differences)
```

Comment:
I hope that the yesterday demonstration has shown clearly that it is not a minute but about two seconds. Of course it may be theoretically longer with some network issues and would be impossible with Bugzilla outage but that is the thing of the design incorporating Bugzilla. SFM2 would has completely same potential issues.

Reply:
Re: just saw your comment below on the overhead of two network calls

Your argument is somewhat valid, it'll be as fast/slow as SFM2 however I still think the WebUI guys will not appreciate it much 😄